2020-12-29 03:40:52 database: wrong user or password; user: "COMPUTINGSERVICE"
org.h2.message.DbException: Wrong user name or password [28000-200]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.engine.Engine.openSession(Engine.java:131)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:309)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:289)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:785)
	at org.h2.server.web.WebApp.login(WebApp.java:1008)
	at org.h2.server.web.WebApp.process(WebApp.java:223)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Wrong user name or password [28000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:461)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 21 more
2020-12-29 03:41:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "USUARIOS" not found; SQL statement:
SELECT * FROM Usuarios [42102-200]
2020-12-29 04:43:51 database: wrong user or password; user: "COMPUTINGSERVICE"
org.h2.message.DbException: Wrong user name or password [28000-197]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.engine.Engine.openSession(Engine.java:107)
	at org.h2.engine.Engine.openSession(Engine.java:179)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:157)
	at org.h2.engine.Engine.createSession(Engine.java:140)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:351)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:103)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:733)
	at org.h2.server.web.WebApp.login(WebApp.java:954)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:134)
	at org.h2.server.web.WebThread.run(WebThread.java:90)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.h2.jdbc.JdbcSQLException: Wrong user name or password [28000-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	... 19 more
2020-12-29 09:20:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO ENGINEERS VALUES (A78A612185C0EBB0EE950513A67C2524, LIGETI@[*]LABOPROGSEGURA.UNAVARRA.ES, A) "; expected "., (, [, ::, *, /, %, +, -, ||, ~, !~, NOT, LIKE, ILIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"; SQL statement:
INSERT INTO ENGINEERS VALUES (a78a612185c0ebb0ee950513a67c2524, ligeti@LABOPROGSEGURA.UNAVARRA.ES, A) [42001-197]
2020-12-29 09:20:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO ENGINEERS VALUES (A78A612185C0EBB0EE950513A67C2524, LIGETI@[*]LABOPROGSEGURA.UNAVARRA.ES, A) "; expected "., (, [, ::, *, /, %, +, -, ||, ~, !~, NOT, LIKE, ILIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"; SQL statement:
INSERT INTO ENGINEERS VALUES (a78a612185c0ebb0ee950513a67c2524, ligeti@LABOPROGSEGURA.UNAVARRA.ES, A) [42001-197]
2020-12-29 09:20:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO ENGINEERS VALUES (A78A612185C0EBB0EE950513A67C2524, LIGETI@[*]LABOPROGSEGURA.UNAVARRA.ES, A) "; expected "., (, [, ::, *, /, %, +, -, ||, ~, !~, NOT, LIKE, ILIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"; SQL statement:
INSERT INTO ENGINEERS VALUES (a78a612185c0ebb0ee950513a67c2524, ligeti@LABOPROGSEGURA.UNAVARRA.ES, A) [42001-197]
2020-12-29 09:22:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "a78a612185c0ebb0ee950513a67c2524" not found; SQL statement:
INSERT INTO ENGINEERS VALUES ("a78a612185c0ebb0ee950513a67c2524", "ligeti@LABOPROGSEGURA.UNAVARRA.ES", "A") [42122-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:150)
	at org.h2.command.dml.Insert.prepare(Insert.java:314)
	at org.h2.command.Parser.prepareCommand(Parser.java:283)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:1077)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1386)
	at org.h2.server.web.WebApp.query(WebApp.java:1060)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1022)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1009)
	at org.h2.server.web.WebThread.process(WebThread.java:165)
	at org.h2.server.web.WebThread.run(WebThread.java:90)
	at java.lang.Thread.run(Thread.java:748)
2020-12-29 09:22:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "A78A612185C0EBB0EE950513A67C2524" not found; SQL statement:
INSERT INTO ENGINEERS VALUES (a78a612185c0ebb0ee950513a67c2524, "ligeti@LABOPROGSEGURA.UNAVARRA.ES", "A") [42122-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:150)
	at org.h2.command.dml.Insert.prepare(Insert.java:314)
	at org.h2.command.Parser.prepareCommand(Parser.java:283)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:1077)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1386)
	at org.h2.server.web.WebApp.query(WebApp.java:1060)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1022)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1009)
	at org.h2.server.web.WebThread.process(WebThread.java:165)
	at org.h2.server.web.WebThread.run(WebThread.java:90)
	at java.lang.Thread.run(Thread.java:748)
